examples/next-prisma-todomvc/prisma/migrations/20230322185952_/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- RenameIndex
+ALTER INDEX "Task.createdAt_unique" RENAME TO "Task_createdAt_key";
```

Comment:
got this when starting dev - have we changed the schema without running migrations earlier??